functorch/functorch/csrc/BatchRulesDecompositions.cpp

```diff
@@ -233,6 +233,7 @@ TORCH_LIBRARY_IMPL(aten, FT_BATCHED_KEY, m) {
   OP_DECOMPOSE2(vsplit, int);
   OP_DECOMPOSE2(vsplit, array);
   OP_DECOMPOSE(vstack);
+  OP_DECOMPOSE(where);
```

Comment:
@ngimel I think you need to decompose the specific overloads. So... looks like

```suggestion
  OP_DECOMPOSE2(where, ScalarOther);
  OP_DECOMPOSE2(where, ScalarSelf);
```